docs/tutorial/quick-start.md

```diff
@@ -11,26 +11,18 @@ It doesn't mean atom-shell is a JavaScript binding to GUI libraries. Instead,
 atom-shell uses web pages as its GUI, so you could also see it as a minimal
 Chromium browser, controlled by JavaScript.
 
-### The browser side
+### The main process
 
-If you have experience with Node.js web applications, you will know that there
-are two types of JavaScript scripts: the server side scripts and the client side
-scripts. Server-side JavaScript is that which runs on the Node.js
-runtime, while client-side JavaScript runs inside the user's browser.
+The main atom-shell process displays a GUI by showing web pages. We have
+**scripts that run in the atom-shell runtime**, that create **scripts
+that run in the web page**. We call them **main process scripts**,
+and **renderer process scripts**.
```

Comment:
I think we can say main process creates web pages, and web pages run in their own renderer processes. We can ignore the "renderer process scripts" part to make things simpler.